### PR TITLE
Update .gitignore and Add limited delta status on cli

### PR DESCRIPTION
**Pull request by ankit2001**
Signed-off-by: ankit2001 <ankitchoudhary202.ac@gmail.com>

### CODE REVIEW

**Comment by ankit2001** [comment]
I have created new pr @majurg , please review it . 
The last one was still pending after passing all test , i do not know why it was not upgrading,
So that's a new pr

File: .gitignore [hunk 1]
```diff
@@ -11,6 +11,7 @@
 /Scripts
 /Lib
 /pip-selfcheck.json
+/output.json
```
Comment (steven-esser) [inline .gitignore]: We do not want this in our gitignore

**Review by steven-esser — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (steven-esser) [reply .gitignore]: You did not remove this yet.

**Review by steven-esser — CHANGES_REQUESTED** [review]

**Comment by ankit2001** [comment]
Ohh sorry, i was out the station today, i will do it asap

**Comment by ankit2001** [comment]
@majurg ,i have done the required changes.

File: src/deltacode/cli.py [hunk 2]
```diff
@@ -55,6 +55,12 @@ def write_json(deltacode, outfile, all_delta_types=False):
     # TODO: add toggle for pretty printing
     simplejson.dump(results, outfile, iterable_as_array=True, indent=2)
     outfile.write('\n')
+   
+    # Output delta_stats at cli
+    status = deltacode.stats.to_dict()
+    click.echo("\nDelta stats :\n")
+    for stat, value in status.iteritems():
+        click.echo("{} : {}".format(stat,value))
```
Comment (steven-esser) [inline src/deltacode/cli.py:63]: Can we break down the stats to handle `factors` as well?

**Review by steven-esser — CHANGES_REQUESTED** [review]

File: src/deltacode/cli.py (hunk 2 above)
Comment (ankit2001) [reply src/deltacode/cli.py:63]: what factors u want @majurg, can u give me some examples??

**Comment by steven-esser** [comment]
Closing due to stagnation.

**Pull request closed without merging**